README.md

```diff
@@ -12,96 +12,96 @@ Pull requests are welcome. Please keep the list in alphabetical order.
-[Envato](https://envato.com/)                     | Sites for creatives                      | Ruby, Web                                      | ✓                                    | ?                       | :keyboard: [Jobs](https://envato.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Envato-EI_IE700841.11,17.htm)
-[Eyeo](https://eyeo.com)                          | AdBlock Plus & More                      | Many                                           | ?                                    | ?                       | :keyboard: [Jobs](https://eyeo.com/jobs/)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-Eyeo-EI_IE848298.11,15.htm)
-[Fairwinds](https://www.fairwinds.com/)           | Kubernetes Services and Software         | Go, Kubernetes, Terraform                      | ✓                                    | ?                       | :keyboard: [Jobs](https://www.fairwinds.com/careers)
-[Fire Engine Red](http://fire-engine-red.com/)    | Education market solutions               | ?                                              | ?                                    | ?                       | :keyboard: [Jobs](http://fire-engine-red.com/about/#careers)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Fire-Engine-RED-EI_IE1339735.11,26.htm)
-[Four Kitchens](http://www.fourkitchens.com/)     | Web consulting                           | Drupal, Node, PHP, Web                         | ?                                    | ?                       | :keyboard: [Jobs](http://www.fourkitchens.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Four-Kitchens-EI_IE588769.11,24.htm)
-[Genuitec](http://www.genuitec.com)               | Eclipse-based tools                      | Java, Eclipse, Web, Mobile                     | ?                                    | ?                       | :keyboard: [Jobs](http://www.genuitec.com/company/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Genuitec-EI_IE346957.11,19.htm)
+[Elastic](https://www.elastic.co/)                | Elasticsearch as a service               | Java, DevOps, Web                              | ✓                                    | Many positions are flexible | :keyboard: [Jobs](https://www.elastic.co/about/careers)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Elastic-EI_IE751551.11,18.htm)
+[Envato](https://envato.com/)                     | Sites for creatives                      | Ruby, Web                                      | ✓                                    | Limited roles, unable to find remote positions | :keyboard: [Jobs](https://envato.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Envato-EI_IE700841.11,17.htm)
```

Comment:
"Work from anywhere" is listed as their top benefit. :slightly_smiling_face:

Reply:
Haha I realise that I am much less trusting of what companies say, some companies say Anywhere but mean "Anywhere in countries where we have offices".

Will update it to Anywhere anyhow, thanks!